code/modules/projectiles/guns/projectile/automatic/z8.dm

```diff
@@ -1,6 +1,6 @@
 /obj/item/weapon/gun/projectile/automatic/z8
 	name = "FS AR 5.56x45mm \"Z8 Bulldog\""
-	desc = "The Z8 Bulldog is an older model bullpup carbine, made by \"Frozen Star\". Uses armor piercing 5.56mm rounds. Makes you feel like a space marine when you hold it."
+	desc = "The Z8 Bulldog is an older model bullpup carbine, made by \"Frozen Star\". Uses 5.56mm rounds."
```

Comment:
Why remove that line?

Reply:
We don't have AP 5.56, only generic ones. Yes, they have higher AP, but you can't notice that even when comparing to AK. And last line moved to Dallas rifle, true weapon of space bald marine.